spec/terminology.md

```diff
@@ -10,10 +10,11 @@
 ~ As defined by [DID Core](https://www.w3.org/TR/did-core/#dfn-did-documents). See section [DID Documents](#did-documents).
 
 [[def: key event log, KEL]]
-~ As defined by KERI.
+~ As defined by [KERI](https://github.com/WebOfTrust/WOT-terms/wiki/key-event-log).
 
 [[def: transaction event log, TEL]]
-~ As defined by KERI.
+~ As defined by [KERI](https://github.com/WebOfTrust/WOT-terms/wiki/transaction-event-log).
```

Comment:
I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

Reply:
> I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

Ouch, I normally do, have to look into what went wrong.

Reply:
I default commit as 'verified'. 
<img width="1328" alt="Screenshot 2023-10-06 at 08 49 36" src="https://github.com/trustoverip/tswg-did-method-webs-specification/assets/479356/da6d942a-e00a-4007-8ed5-4f9251b479d5">

I see that DCO is something else / extra. I'll go and try to add this data to my commits

Reply:
Done

Reply:
> I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

Done, great point: stay in the spec.